path-manager/app/model/PathRecord.scala

```diff
@@ -16,8 +16,8 @@ object PathRecord {
 
   def apply(item: Item): PathRecord = PathRecord(
     path = item.getString("path"),
-    `type` = item.getString("type"),
     identifier = item.getLong("identifier"),
+    `type` = item.getString("type"),
```

Comment:
any reason for these changing order?

Reply:
Yeah, to match the case class order- this specific ordering doesn't matter but doesn't match. (This is from an abandoned refactoring that destructured the case class- and this had me confused over field order)